app/views/studies/_description.html.erb

```diff
@@ -0,0 +1,15 @@
+<%
+  setting = Array(settings.filter { |s| s.attribute_key == "simple_description" }).first
+  page = controller.action_name == "index" ? "list" : controller.action_name
+  display = !!setting.attributes["display_on_#{page}"]
+  display_label = !!setting["display_label_on_#{page}"]
```

Comment:
@johnsja these are gross, I know. As we adjust some these elements to make use of better semantic markup, I think we're likely to reach more for partials instead of the generic helper. When/if we change a few more I'd like to make some changes to `TrialAttributeSetting` that'll make these go away. For now,  🤮 .